docs/UserGuide.md

```diff
@@ -216,7 +216,7 @@ Task successfully created.
         Task have yet to be assigned to anyone
 ```
 
-#### 3.3.2. View Task(s)
+#### 3.3.2. <a id="view-task">View Task(s)</a>
```

Comment:
Mj, can help make this consistent?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

Reply:
Ok I've fixed the inconsistencies.